core/src/execution/constants.rs

```diff
@@ -3,3 +3,5 @@ pub const PARALLEL_QUERY_BATCH_SIZE: usize = 20;
 // We currently limit the max number of logs to fetch,
 // to avoid blocking the client for too long.
 pub const MAX_SUPPORTED_LOGS_NUMBER: usize = 5;
+
+pub const MAX_STATE_HISTORY_LENGTH: usize = 64;
```

Comment:
@ncitron; this should be 256 or 64? The previous 64 value was not being used anywhere.

Reply:
Could also make sense to have different values for Ethereum / OP.

Reply:
64 is probably fine for now. Maybe would want to make this configurable though in the future.